mender/templates/_image-pull-secret.yaml

```diff
@@ -1,3 +1,3 @@
-{{- define "imagePullSecret" }}
+{{- define "mender.enterprisePullSecret" }}
```

Comment:
I chose this way: this template is used to define the imagePullSecrets for enterprise images only, so I just renamed it, to avoid confusions.